qa-test3.planx-pla.net/portal/gitops.json

```diff
@@ -48,8 +48,8 @@
     "appName": "NIAID Data Ecosystem",
     "index": {
       "introduction": {
-        "heading": "NIAID Data Ecosystem Datasets",
-        "text": "Disciplines within NIAID can have their own independent data models while maintaining interoperability of the data ecosystem. The data within projects are findable, accessible, interoperable, and reusable (FAIR), but it will be easier for each individual research community to work within the system.", 
+        "heading": "Cross Environment Datasets",
+        "text": "Divisions within NIAID can have their own independent data models while maintaining interoperability of the data ecosystem. The data within projects are findable, accessible, interoperable, and reusable (FAIR), but it will be easier for each individual research community to work within the system. Of course, with this approach, two different research communities may create data models that have data elements that may conflict and that will need to be manually de-conflicted.",
```

Comment:
any reason for changing this? I know this is QA env but these texts are not approved. Maybe we should keep this part the same as prod